src/commands/aksCreateCluster/aksCreateClusterFromCopilot.ts

```diff
@@ -1,6 +1,69 @@
 import { IActionContext } from "@microsoft/vscode-azext-utils";
 import * as vscode from "vscode";
+import { getReadySessionProvider } from "../../auth/azureAuth";
+import { failed } from "../utils/errorable";
+import { getSubscriptions, handleNoSubscriptionsFound, SelectionType, SubscriptionQuickPickItem } from "../utils/subscriptions";
+import { window } from "vscode";
+import { checkExtension, handleExtensionDoesNotExist } from "../utils/extensions";
+import { GITHUBCOPILOT_FOR_AZURE_VSCODE_ID } from "../../plugins/shared/constants";
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
💡 Lets avoid silencing these things :) linting helps in well written code and it helps in latter in grading the code which happens at the repo css level. 

lets remove this and make proper type returns.

Reply:
this is now updated